app/app.rb

```diff
@@ -0,0 +1,49 @@
+ENV['RACK_ENV'] ||= "development"
+require "sinatra/base"
+require "sinatra/flash"
+require "sinatra/partial"
+require_relative "models/peep"
+require_relative "models/user"
+require_relative "models/data_mapper_setup"
+require_relative "controllers/peeps"
+require_relative "controllers/users"
+require_relative "controllers/sessions"
+
+data_mapper_configure
```

Comment:
Can you put this method in a module in `data_mapper_setup` so you can call it with `DataMapperSetup.data_mapper_configure`.  This namespaces the method to make it clear what section of the app it belongs to.